docs/user-guide/workflow.ipynb

```diff
@@ -164,9 +167,12 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "he3_workflow = pol.he3.He3CellWorkflow(in_situ=False)\n",
+    "he3_polarizer_workflow = pol.He3CellWorkflow(in_situ=False, incoming_polarized=False)\n",
+    "he3_analyzer_workflow = pol.He3CellWorkflow(in_situ=False, incoming_polarized=True)\n",
```

Comment:
Note that the polarizer is always treated in the same way, since the `incoming_polarized` only affects the analyzer, so there is no need to split this.

Reply:
Yes, I agree. Dont you think that anyhow also here splitting it gives a better description for readers to know what is used where, and then again in an additional text we can mention about the fact that the polarized-incoming beam is only considered for the analyzer here?

Reply:
Maybe this indicates that the current naming `incoming_polarized` is unclear?

Reply:
Hm, I would not say so...rather that users might forget that it doesnt make sense to have an incoming polarized beam for the polarizer, as there is nothing to polarize them before that. But to have a better description in might be good then.